File: src/test/java/io/github/dmitrikudrenko/calculator/CalculatorApplication.java
package io.github.dmitrikudrenko.calculator;

import io.appium.java_client.AppiumDriver;
import io.github.dmitrikudrenko.core.AbstractApplication;

public class CalculatorApplication extends AbstractApplication {
    public CalculatorApplication(AppiumDriver driver) {
        super(driver);
    }

    public CalculatorScreen calculatorScreen() {
        return new CalculatorScreen(driver);
    }
}

File: src/test/java/io/github/dmitrikudrenko/calculator/core/CalculatorApplication.kt
package io.github.dmitrikudrenko.calculator.core

import io.appium.java_client.AppiumDriver
import io.github.dmitrikudrenko.core.AbstractApplication

class CalculatorApplication(driver: AppiumDriver<*>) : AbstractApplication(driver) {

    fun calculatorScreen(): CalculatorScreen {
        return CalculatorScreen(driver)
    }
}
